GettingStarted/ibmdbpy_GettingStarted_2-geo_basics.ipynb

```diff
@@ -0,0 +1,1621 @@
+       "      <td>-107.8397178</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>5</td>\n",
+       "      <td>MULTIPOLYGON (((-91.2589262966 36.2578866492, ...</td>\n",
+       "      <td>05</td>\n",
+       "      <td>121</td>\n",
+       "      <td>00069178</td>\n",
+       "      <td>Randolph</td>\n",
+       "      <td>05121</td>\n",
+       "      <td>Randolph County</td>\n",
+       "      <td>06</td>\n",
+       "      <td>H1</td>\n",
+       "      <td>G4020</td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td></td>\n",
+       "      <td>A</td>\n",
+       "      <td>1.689166e+09</td>\n",
+       "      <td>9968439.0</td>\n",
+       "      <td>+36.3412985</td>\n",
+       "      <td>-091.0284409</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   OBJECTID                                              SHAPE STATEFP  \\\n",
+       "0         1  MULTIPOLYGON (((-99.4756582604 33.8340108094, ...      48   \n",
+       "1         2  MULTIPOLYGON (((-96.6219873342 30.0442882117, ...      48   \n",
+       "2         3  MULTIPOLYGON (((-99.4497297204 46.6316377481, ...      38   \n",
+       "3         4  MULTIPOLYGON (((-107.4817473750 37.0000108736,...      08   \n",
+       "4         5  MULTIPOLYGON (((-91.2589262966 36.2578866492, ...      05   \n",
+       "\n",
+       "  COUNTYFP  COUNTYNS       NAME  GEOID          NAMELSAD LSAD CLASSFP  MTFCC  \\\n",
+       "0      487  01384029  Wilbarger  48487  Wilbarger County   06      H1  G4020   \n",
+       "1      015  01383793     Austin  48015     Austin County   06      H1  G4020   \n",
+       "2      047  01035620      Logan  38047      Logan County   06      H1  G4020   \n",
+       "3      067  00198148   La Plata  08067   La Plata County   06      H1  G4020   \n",
+       "4      121  00069178   Randolph  05121   Randolph County   06      H1  G4020   \n",
+       "\n",
+       "  CSAFP CBSAFP METDIVFP FUNCSTAT         ALAND      AWATER     INTPTLAT  \\\n",
+       "0        46900                 A  2.514474e+09  18257915.0  +34.0849199   \n",
+       "1   288  26420                 A  1.674401e+09  25610780.0  +29.8919013   \n",
+       "2                              A  2.571371e+09  47715597.0  +46.4692780   \n",
+       "3        20420                 A  4.382664e+09  19545452.0  +37.2873673   \n",
+       "4                              A  1.689166e+09   9968439.0  +36.3412985   \n",
+       "\n",
+       "       INTPTLON  \n",
+       "0  -099.2424397  \n",
+       "1  -096.2701696  \n",
+       "2  -099.5045846  \n",
+       "3  -107.8397178  \n",
+       "4  -091.0284409  "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "idageodf.head()\n",
+    "# what about this warning ?"
```

Comment:
That is a very good question! 😄

Reply:
well I just remove the comment, the warning is on JDBC / Db2 level, not ibmdbpy.